package.json

```diff
@@ -9,15 +12,9 @@
     "@guardian/eslint-config": "file:packages/eslint-config",
     "@guardian/eslint-config-typescript": "file:packages/eslint-config-typescript",
     "@guardian/prettier": "file:packages/prettier",
-    "@typescript-eslint/eslint-plugin": "^4.26.0",
-    "@typescript-eslint/parser": "^4.26.0",
-    "eslint": "^7.28.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-import": "^2.23.4",
-    "eslint-plugin-prettier": "^3.4.0",
+    "eslint": "8.0.0",
     "lerna": "^4.0.0",
-    "prettier": "^2.3.1",
-    "typescript": "^4.3.2"
+    "prettier": "2.4.0",
```

Comment:
Nit: is there a reason we must pin to this specific version of `prettier`, or can we specify a `^` range in keeping with  other devDeps?

Reply:
2.4 has the new brackets flag config, so this becomes the minimum version the workspace should work with

Reply:
I get ya. So the general rule is:

- in peerDependencies, specify the minimum supported version of a package, using a `^` to pin the major version and enforce a minimum minor version
- in devDependencies,  specify the minimum supported version of a package, using the exact version (i.e. no `^` or `~`)

Reply:
that's was the thinking yeah – make it as flexible as possible to consume, but make sure we develop with the smallest possible feature set we need. and then obvs if we need a feature from a later version that becomes the baseline